.github/workflows/rss-runner.yaml

```
@@ -0,0 +1,33 @@
+  workflow_dispatch:
+  
+jobs:
+  run-RSS-Runner:
+    runs-on: ubuntu-latests
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+      
+      - name: Set up Python
+        uses: actions/setup-python@v3
```

actionlint (1.7.7)

the runner of "actions/setup-python@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)